catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java

```diff
@@ -70,9 +70,12 @@
 
     private FilterBuilder filterBuilder;
 
+    private boolean removeSourceLibrary;
+
     private static final Logger LOGGER = LoggerFactory.getLogger(BqsConverter.class);
 
-    public BqsConverter(FilterBuilder filterBuilder) {
+    public BqsConverter(FilterBuilder filterBuilder, boolean removeSourceLibrary) {
+        this.removeSourceLibrary = removeSourceLibrary;
         this.filterBuilder = filterBuilder;
     }
 
```

Comment:
These values are used later without null checks.  You should verify that they aren't null either here, or where they're used if null is a valid value.